manager/src/test/java/com/redhat/service/bridge/manager/workers/resources/ConnectorWorkerTest.java

```diff
@@ -88,6 +89,7 @@ void handleWorkProvisioningWithUnknownResource() {
         assertThatCode(() -> worker.handleWork(work)).isInstanceOf(IllegalStateException.class);
     }
 
+    @Transactional
```

Comment:
I wonder why is this needed now?

Reply:
Because I've made `WorkManager.schedule(..)` demand a transaction (as work should only be scheduled in the same transaction as the resource is created.. failure to create the resource or work, for whatever reason, should rollback both). The tests use the DAO directly (not services to create/delete bridges/processors) so need now to become transactional for their use of `WorkManger.schedule(..)` to function.